ui/create_restore_page.go

```diff
@@ -629,11 +772,23 @@ func (pg *createRestore) centralize(gtx layout.Context, content layout.Widget) l
 	)
 }
 
+func (pg *createRestore) computePasswordStrength(common pageCommon, editors ...*widget.Editor) {
+	password := editors[0]
+	strength := dcrlibwallet.ShannonEntropy(password.Text()) / 4.0
+	pg.passwordStrength.Progress = float32(strength * 100)
+	pg.passwordStrength.Color = common.theme.Color.Success
+}
```

Comment:
This method is also used in modal_templates.go. It can be added to the ui/util.go file and used by both modals and restore pages. This would eliminate duplication.